app/lib/efile/ny/it201.rb

```diff
@@ -60,19 +60,19 @@ def calculate
         set_line(:IT201_LINE_40, :calculate_line_40)
         set_line(:IT201_LINE_43, :calculate_line_43)
         set_line(:IT201_LINE_44, :calculate_line_44)
-        set_line(:IT201_LINE_45, :calculate_line_45)
```

Comment:
this was a mistake i made on an earlier ticket, applying arizona logic to this ny form. we aren't supporting this line on the ny form so removing it.